src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -154,6 +156,28 @@ private void executeTestConfigurationNoGuaranteedCleanup() throws Exception {
       logger.info("Version: Skipping version determination");
     }
 
+    if (config.server.testRunnerEnvironmentScript != null) {
+      TestRunnerEnvironmentScript ghContextScript =
+          config
+              .server
+              .testRunnerEnvironmentScript
+              .scriptClass
+              .getDeclaredConstructor()
+              .newInstance();
+      gitHubContextResult = ghContextScript.getTestRunnerEnvironmentContext();
+    } else {
+      try {
+        Class<?> scriptClassGeneric = GitHubActionsWorkflowRunContext.class;
```

Comment:
Why is running the `GitHubActionsWorkflowRunContext` script the default for servers which haven't specified a script instead of something like not running a script at all?

Reply:
My thoughts are this will save ourselves from adding additional configs to the test repo.